[WIP] checks-run: remove duplicate `coverage.rc` file creation

cmoussa1 opened the pull request:
#### Problem

`checks_run.sh` has two instances of creating a `coverage.rc` file.

---

This PR removes the duplicate creation of `coverage.rc`. Also looking to see if this might fix the missing Python bindings coverage for this repository.

cmoussa1 committed:
codecov: remove duplicate coverage.rc creation

Problem: checks_run.sh has two instances of creating a coverage.rc
file.

Remove the duplicate creation of coverage.rc.

cmoussa1 commented:
No, not a problem. I opened this PR mostly as an experiment to see if removing the duplicate `coverage.rc` that I had in flux-accounting's `checks_run.sh` (I think due to my my negligence in #644) would actually trigger the Python bindings from actually getting picked up by Codecov, but I'm still not seeing coverage there. 🤷

grondo commented:
I wonder if this error is the problem:
```
  No source for code: '/usr/src/usr/src/src/bindings/python/fluxacct/accounting/__init__.py'.
```

I'm not sure why the extra `/usr/src`, but maybe you can see if you add `ignore_errors = True` to the `coverage.rc` if you can get past that?

cmoussa1 committed:
t: remove duplicate pycotap/ directory

Problem: The pycotap/ directory is mistakenly duplicated both inside the
t/ directory and inside the t/python/tap/ directory. It should follow
flux-core's structure, which just contains pycotap/ inside of
t/python/tap/.

Remove the duplicate directory.

Update the PYTHONPATH variable in t/Makefile.am to account for the
change.

Add a PY_LOG_DRIVER variable to t/Makefile.am to match flux-core's
t/Makefile.am.
